src/v3/README.md

```diff
@@ -1,12 +1,152 @@
+
+The third generation Sign-In Widget doesn't support embedded authentication.
+
+See [Okta deployment models - redirect vs. embedded](https://developer.okta.com/docs/concepts/redirect-vs-embedded/).
+
+### Authenticators
+
+The third generation Sign-In Widget doesn't currently support these authenticators:
+
+- Duo Security
+- RSA
+- Symantec VIP
+- IdP authenticator
+- Social login
+- Custom Authenticator
+
+### Branding
+
+The third generation Sign-In Widget makes multibrand customization easier, but it doesn't support custom CSS overrides. If your org uses the code editor to customize a sign-in page, those customizations don't migrate. See [Style the sign-in page](https://developer.okta.com/docs/guides/custom-widget/main/).
+
+## Compare generations
+
+| Feature | Second generation | Third generation |
+| ---                | ---                            | ---             |
+| Deployment | Redirect (Okta-hosted)</br>Embedded (self-hosted)</br>Pinning to a specific Okta-hosted version | Redirect (Okta-hosted)</br>Automatic deployment of latest EA version |
+| Authenticators | Email</br>Password</br>Okta Verify (Okta FastPass, push, TOTP)</br>WebAuthn</br>Smart Card IdP</br>Google Authenticator</br>YubiKey OTP</br>SMS</br>Security Question</br>Custom</br>Duo Security</br>RSA</br>Symantec VIP</br>IdP authenticator</br>Social login | Email</br>Password</br>Okta Verify (Okta FastPass, push, TOTP)</br>WebAuthn</br>Smart Card IdP</br>Google Authenticator</br>YubiKey OTP</br>SMS</br>Security Question |
+| Flows | Profile enrollment</br>Authenticator enrollment</br>Authenticator verification</br>Forgot password</br>Account unlock | Profile enrollment</br>Authenticator enrollment</br>Authenticator verification</br>Forgot password</br>Account unlock |
+| Globalization | Right-to-left languages (requires customization)</br>All out-of-the-box translations</br>String customization</br>Bring your own language | All out-of-the-box translations</br>String customization</br>Bring your own language |
+| Branding | Branding and multibrands</br>CSS overrides (code editor) | Branding and multibrands |
+
+
+## Type of Applications
+
+### OIDC
+
+* SPA Introduction
+* Web Introduction
+
+### Non-OIDC
+
+* SAML?
+
+
+## Okta deployment models
+
+### Redirect (Okta-hosted)
+
+A user sign-in flow that grants authentication control to Okta by redirecting to an Okta hosted sign-in page using open protocols like OAuth 2.0 and SAML. See [Redirect authentication](https://developer.okta.com/docs/guides/redirect-authentication/).
+
+### Embedded (self-hosted)
+
+The third generation Sign-In Widget doesn't support embedded authentication. See [Embedded (self-hosted), second generation](https://github.com/okta/okta-signin-widget#embedded-self-hosted).
+
+## Migrating to the third generation
+
```

Comment:
@lesterchoi-okta  How much of the migration content should reside here? What's required?